package.json

```diff
@@ -55,20 +55,20 @@
     "@types/cors": "^2.8.4",
     "@types/express": "^4.16.1",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^10.14.1",
-    "@types/sequelize": "^4.27.46",
-    "@types/yargs": "^12.0.8",
+    "@types/node": "^10.14.4",
+    "@types/sequelize": "^4.27.47",
```

Comment:
This dependency can be removed because `sequelize` does have native support for TypeScript since `v5`